### PR TITLE
Done. Added a checkbox to force the adv indicator on skills/talent

### PR DESCRIPTION
**Pull request by JiDW**
image

### CODE REVIEW

File: scripts/item/item-sheet.js [hunk 1]
```diff
@@ -174,6 +174,8 @@ class ItemSheetWfrp4e extends ItemSheet
 
     data.showBorder = data.item.img == "systems/wfrp4e/icons/blank.png" || !data.item.img
     data.isGM = game.user.isGM;
+    data.isOwned = this.item.isOwned;
+    data.flags = this.item.data.flags;
```
Comment (moo-man) [inline scripts/item/item-sheet.js]: Pretty sure these can be accessed already, have you investigated that?

**Comment by moo-man** [comment]
Looks good, could you add a tooltip for the checkbox (include the whole div so you don't need to hover specifically on the checkbox)?

File: scripts/item/item-sheet.js (hunk 1 above)
Comment (JiDW) [reply scripts/item/item-sheet.js]: Yes, I mean I tried to find them by looking at the 'data' variable in chrome inspector but I could have missed them if they are hidden deeper than I explored. I'll take another look tonight

**Comment by JiDW** [comment]
Ok, will do!

File: scripts/item/item-sheet.js (hunk 1 above)
Comment (moo-man) [reply scripts/item/item-sheet.js]: I could not find isOwned (it's a getter, not actual item data), but inside `data` is `item.flags`, so you can access that in handlebars with just `{{item.flags}}`

**Pull request merged**